json-fleece-core/package.yaml

```diff
@@ -1,5 +1,5 @@
 name:                json-fleece-core
-version:             0.5.1.0
+version:             0.5.1.1
```

Comment:
I guess this should really be `0.5.2.0` since it introduces new API

Reply:
Which versioning scheme are we doing anyway? As far as I know, PVP says that in A.B.C, A.B is the major version. If an API addition is made, this part must be incremented:

>  Breaking change. If any entity was removed, or the types of any entities or the definitions of datatypes or classes were changed, or orphan instances were added or any instances were removed, then the new A.B MUST be greater than the previous A.B. Note that modifying imports or depending on a newer version of another package may cause extra orphan instances to be exported and thus force a major version change.

Reply:
@ysangkok I am not drawing the same conclusion as you from that language. An API addition doesn't represent a breaking change.